kivy_ios/recipes/ffpyplayer/__init__.py

```diff
@@ -3,7 +3,7 @@
 
 
 class FFPyplayerRecipe(CythonRecipe):
-    version = "4.2.0"
+    version = "v4.3.5"
     url = "https://github.com/matham/ffpyplayer/archive/{version}.zip"
     library = "libffpyplayer.a"
     depends = ["python", "ffmpeg"]
```

Comment:
Hi @tito!

Seems that the CI job "build_updated_recipes" is failing as we missed a needed dependency in `depends` list,
This will likely go unnoticed while building it manually but is definitely something nice to fix.

I've added a suggestion here:
  
```suggestion
    depends = ["python", "sdl2", "ffmpeg"]
```